TTS/tts/layers/xtts/tokenizer.py

```diff
@@ -258,10 +258,11 @@ def expand_abbreviations_multilingual(text, lang="en"):
             ("$", " dollar "),
             ("£", " pound "),
             ("°", " degree "),
+            (".", ""),
```

Comment:
> the dot should stay with a space between the word and the dot. then use my patch and everyone is happy to have a TTS not pronouncing "dot" at each sentence.

Ok. I'm sorry, but I don't fully understand your change yet then. With these lines, you _are_ removing the dot, no?

Reply:
sorry I didn't get notifications from your last comment.
with (".", ""), into the list it's indeed stop to pronounce "dot".
btw i should be added to all supported languages.

Reply:
ok my PR seems to affect too much some languages. and btw this is not the right solution.
If I knew at least where it comes from the coqui source code so I will try to solve it. but for now I need to fix my own bugs....

Reply:
@ROBERT-MCDOWELL Yes, strings on the left side are being replaced with those on the right, so this fully removes any periods, which I think is too broad. The issue comes not from the Coqui source code, but from the training data used for XTTS, so there is no obvious way to fix it.

Reply:
is there a way to update into the trainged model rather than recreate a new one from scratch?

Reply:
Not sure what exactly you mean, but fine-tuning XTTS is the only official way anyway, because they never released the full training code.

Reply:
ah ok :(